packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1605,15 +1605,15 @@ describe('ReactDOMInput', () => {
       };
     }
 
-    it('always sets the attribute when values change on text inputs', function() {
+    it('retains the initial value attribute when values change on text inputs', function() {
```

Comment:
It's always been this way, but I don't like how React stores an initial value:
https://github.com/facebook/react/blob/master/packages/react-dom/src/client/ReactDOMFiberInput.js#L25

I think the value should just be blank when it becomes null, or revert to the defaultValue prop (which could be null).